### ISSUE TITLE
Delete resource endpoint appears to use relative pathing

### ISSUE THREAD

**Issue by jeremyjaybaker** [redux-json-api/redux-json-api#165]
I keep getting incorrect URLs for my API endpoint when using `deleteResource`. I checked the source and in `jsonapi.js` on line 153, the endpoint is indeed determined without a leading slash so it will always be relative: 
const endpoint = \`${resource.type}/${resource.id}\`;

Is this intentional? And if so, how can I configure this plugin to always use absolute pathing?

**Comment by egeriis** [comment]
I think this comes from the expectation that a `baseURL` has been provided to Axios (through `setAxiosConfig`. Might not be optimal, what do you think?

**Comment by jeremyjaybaker** [comment]
Hmm, I thought I configured the axiosConfig correctly with the `baseUrl` but I may be wrong. I'll check that now and respond when I figure out what's going on.

I'm assuming you're asking my thoughts on whether setting `baseURL` should be necessary in order to determine the endpoint is optimal? I feel like the `baseURL` should default to something like `window.location.origin`. If this is not the url that the user actually wishes to use, it can still be overridden via the same way `baseURL` is set now.

**Comment by jeremyjaybaker** [comment]
Ok, I had to deal with some CORS issues before I actually addressed this one.

Correctly setting `baseURL` fixed my issue. For some reason I was expecting the `endpoint` param to act like the `baseURL` param. While the naming conventions definitely should've clued me in that I was configuring it wrong, it also seems like the docs are a little out of date? I can't find any of the `set` methods that are listed here anywhere in the repository: https://github.com/redux-json-api/redux-json-api/blob/master/docs/set-up-configure.md#configure-api-endpoints-and-access-token

Regardless, we can close this issue and address the docs separately if that's something you agree with.
